pkg/controller/session/model_convert_test.go

```diff
@@ -153,4 +155,52 @@ var _ = Describe("Basic model conversion", func() {
 		})
 
 	})
+	Context("route to route", func() {
+		var (
+			route model.Route
+		)
+		JustBeforeEach(func() {
+			route = session.RouteToRoute(&sess)
```

Comment:
I think this should belong to the test itself. From my point of view before/after are for test fixtures / setup / cleanup, but not for the interaction with the actual component under test.

Reply:
Just as a note; that would include all the model tests.

Reply:
Right, as it's mainly conversion logic being tested I can think we are not really improving readability by a lot with moving/duplicating that.